Fix lazy-object-proxy version in requirements for centos7

r0x0d opened the pull request:
When running the lint or tests in the CI, the image fails to build because of
the dependency version for `lazy-object-proxy` being v1.7.0, this PR intends
to fix the version to v.1.6.0 as it's the working version.

Output of CI failing because of the dep version:
* https://github.com/oamg/convert2rhel/runs/4534232358?check_suite_focus=true#step:4:686

Output of `pipdeptree`:
```bash
[root@c3e8cf79269c /]# pipdeptree
pylint==1.9.5
  - astroid [required: >=1.6,<2.0, installed: 1.6.6]
    - backports.functools-lru-cache [required: Any, installed: 1.6.4]
    - enum34 [required: >=1.1.3, installed: 1.1.10]
    - lazy-object-proxy [required: Any, installed: 1.6.0]
    - singledispatch [required: Any, installed: 3.7.0]
      - six [required: Any, installed: 1.16.0]
    - six [required: Any, installed: 1.16.0]
    - wrapt [required: Any, installed: 1.13.3]
  - backports.functools-lru-cache [required: Any, installed: 1.6.4]
  - configparser [required: Any, installed: 4.0.2]
  - isort [required: >=4.2.5, installed: 4.3.21]
    - backports.functools-lru-cache [required: Any, installed: 1.6.4]
    - futures [required: Any, installed: 3.3.0]
  - mccabe [required: Any, installed: 0.6.1]
  - singledispatch [required: Any, installed: 3.7.0]
    - six [required: Any, installed: 1.16.0]
  - six [required: Any, installed: 1.16.0]
```

Output of `pip show lazy-object-proxy`:
```bash
[root@c3e8cf79269c /]# pip show lazy-object-proxy
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
Name: lazy-object-proxy
Version: 1.6.0
Summary: A fast and thorough lazy object proxy.
Home-page: https://github.com/ionelmc/python-lazy-object-proxy
Author: Ionel Cristian Mărieș
Author-email: contact@ionelmc.ro
License: BSD-2-Clause
Location: /usr/lib64/python2.7/site-packages
Requires:
Required-by: astroid
```

Jira reference (If any):
Bugzilla reference (If any):

abadger commented:
Do we know if this is a bug in the 1.7.0 version specifically?  If so we might want to exclude 1.7.0 (or 1.7.x) instead of limiting to 1.6.0.  If upstream thinks this is a valid change in behaviour or we don't know what upstream thinks then I'm fine with limiting the version although maybe we want to limit it to < 1.7 for greater flexibility?

r0x0d commented:
Talking with @abadger we discovered that this PR isn't needed anymore. The provider for `lazy-object-proxy` has released version 1.7.1 which ignores python2.7, so, pip will default the installation to 1.6.0 now. 

From the pypi page of `lazy-object-proxy`:

>Changelog
1.7.1 (2021-12-15)

    Removed most of the Python 2 support code and fixed python_requires to require at least Python 3.6.

    Note that 1.7.0 has been yanked because it could not install on Python 2.7. Installing lazy-object-proxy on Python 2.7 should automatically fall back to the 1.6.0 release now.

The commit that introduced the change: https://github.com/ionelmc/python-lazy-object-proxy/commit/16db739035116ed74af16aff4a8f2cfeab6fce80
